frontend/src/app/(routes)/governance/VotePopup.tsx

```diff
@@ -0,0 +1,143 @@
+'use client';
+import React, { useState } from 'react';
+import Image from 'next/image';
+import './style.css';
+
+const VotePopup = () => {
+  const [voteOption, setVoteOption] = useState<string>('');
+  const [isOpen, setIsOpen] = useState(true);
+
+  const handleVoteChange = (option: string) => {
+    setVoteOption(option);
+  };
+  const handleClose = () => {
+    console.log('closing votePopup');
+    setIsOpen(false);
+  };
+  if (!isOpen) {
+    return null;
+  }
+  return (
+    <div className="popup-grid">
+      <div className="cross" onClick={handleClose}>
+        <Image
+          src="./plainclose-icon.svg"
+          width={24}
+          height={24}
+          className="cursor-pointer"
+          alt="Plainclose-Icon"
+        />
+      </div>
+      <div className="image-grid">
+        <div className="flex">
+          <Image
+            src="/vote-image.png"
+            width={335}
+            height={298}
+            alt="Vote-Image"
+          />
+        </div>
+        <div className="text-grid">
+          <div className="space-y-6">
+            <div className="proposal-text-big">vote</div>
+            <div className="text-form">
+              <div className="space-y-1">
+                <div className="space-x-2 flex">
+                  <Image
+                    src="./cosmos-logo.svg"
+                    width={40}
+                    height={40}
+                    alt="Cosmos-Logo"
+                  />
+                  <p className="proposal-text-small">#123 | Proposal</p>
+                </div>
+                <div className="proposal-text-normal">
+                  Adjust Trade and Earn Rewards Margined Protocol
+                </div>
+                <div className="proposal-text-small">Voting ends in 2 days</div>
+              </div>
+            </div>
+            <div className="flex w-full justify-between">
+              <div className="radio-buttons">
+                <label className="radio-container">
+                  <div className="flex space-x-2">
+                    <input
+                      type="radio"
+                      name="voteOption"
+                      value="yes"
+                      checked={voteOption === 'yes'}
+                      onChange={() => handleVoteChange('yes')}
+                    />
+                    <span className="radio-checkmark">
+                      <span className="radio-check"></span>
+                    </span>
+                    Yes
+                  </div>
+                </label>
```

Comment:
Create a component for this and pass voteOption, handleVoteChange function and other required props to the component.
Use the same component in next places